pkgs/tools/filesystems/sandboxfs/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, fetchFromGitHub
+, rustPlatform
+, fuse
+, pkg-config
+, installShellFiles
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "sandboxfs";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "bazelbuild";
+    repo = pname;
+    rev = "sandboxfs-${version}";
+    sha256 = "Ia6rq6FN4abnvLXjlQh4Q+8ra5JThKnC86UXC7s9//U=";
+  };
+
+  cargoSha256 = "sha256-k303TjWG+n+/ZMmko84KJtYb7swuQ1ZJOc4Vq6aOhX0=";
```

Comment:
I always forget if we already permit SRI hashes, but from a quick grep it seems that we have a lot of them already.

Reply:
I computed these hashes by setting these fields to the empty string and then copy-pasting the hash in the error message from nix-build. This is the format that nix-build used to print them. I'd be happy to switch to a different format if that's preferable.